### ISSUE TITLE
"uncontrolled noise" check in calibration a little overzealous?

### ISSUE THREAD

**Issue by klapo** [dtscalibration/python-dts-calibration#82]
When attempting to run the `calibration_single_ended` on the same dataset briefly shown in issue #81, I trigger the uncontrolled noise error:
```
ultima_flyfox_raw_ds.calibration_single_ended(st_label='Ps',
                                              ast_label='Pas',
                                              method='ols')
```
yields
```
---------------------------------------------------------------------------
AssertionError                            Traceback (most recent call last)
<ipython-input-44-99ef32957281> in <module>
      1 ultima_flyfox_raw_ds.calibration_single_ended(st_label='Ps',
      2                                               ast_label='Pas',
----> 3                                               method='ols')

/anaconda/envs/dts-calibration/lib/python3.7/site-packages/dtscalibration/datastore.py in calibration_single_ended(self, sections, st_label, ast_label, st_var, ast_var, store_c, store_gamma, store_dalpha, store_alpha, store_tmpf, store_p_cov, store_p_val, variance_suffix, method, solver, p_val, p_var, p_cov, fix_gamma, fix_dalpha)
   1423         assert not np.any(
   1424             self[st_label] <= 0.), \
-> 1425             'There is uncontrolled noise in the ST signal'
   1426         assert not np.any(
   1427             self[ast_label] <= 0.), \

AssertionError: There is uncontrolled noise in the ST signal
```
The issue is that the internal coil of the ultima allows for non-positive stokes and anti-stokes values, as shown below.
![image](https://user-images.githubusercontent.com/6980561/74752134-85a0a780-526e-11ea-94d2-e2cac2d4f131.png)

I would suggest changing the check to only look at positive LAF (or x) values so that someone doesn't naively trigger the error and the phrase "uncontrolled noise" would be more descriptive.

**Comment by bdestombe** [comment]
You're right. It should only check for smaller than zero values along the reference sections.

Thank you for helping out by posting your issues, please keep them coming ;)

**Comment by bdestombe** [comment]
Hi klapo,
This should be fixed with #97 . Feel free to reopen the issue if this not the case
